Classes/Controller/MetadataController.php

```diff
@@ -142,6 +142,18 @@ protected function printMetadata(array $metadata, $useOriginalIiifManifestMetada
                 foreach ($section as $name => $value) {
                     // NOTE: Labels are to be escaped in Fluid template
 
+                    $metadata[$i][$name] = is_array($value)
+                        ? implode($this->settings['separator'], $value)
+                        : $value;
+
+                    // TODO: This might not be necessary for subentries or needs to be typecasted correctly.
```

Comment:
Are these TODOs still relevant? The two-level metadata feature should be implemented completely. So either these TODOs are done (and should be removed) or should be done.